### PR TITLE
add clusters tab as an nbextension

### PR DESCRIPTION
**Pull request by minrk**
related to jupyter/notebook#121

Load with

```python
c.NotebookApp.server_extensions.append('ipyparallel.nbextension')
```

### CODE REVIEW

**Comment by Carreau** [comment]
+1. 

Might need to cleanup the require(namespace) at some point later.

**Comment by Carreau** [comment]
Should re-add the tab when extension is not enabled that link to instruction on ipyparallel, at least for 4.0 not to confuse people. Marking as waiting for author.

**Comment by Carreau** [comment]
I suppose you could also merge `{ "NotebookApp" :{ "server_extensions": ["ipyparallel.nbextension"]}}` with the notebook config with a CLI to install the extension. 
Just though. But works as advertized. 

+1

**Comment by minrk** [comment]
Added an installation note to README

File: ipyparallel/nbextension/clustermanager.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+            def init_signal(self):
+                pass
+            def reinit_logging(self):
+                pass
+        
+        starter = DummyIPClusterStart(log=self.log)
+        starter.initialize(['--profile-dir', profile_dir])
+        cl = starter.controller_launcher
+        esl = starter.engine_launcher
```
Comment (jdfreder) [inline ipyparallel/nbextension/clustermanager.py]: Where'd the `s` come from in the acronym for engine launcher? lol

Comment (minrk) [inline ipyparallel/nbextension/clustermanager.py]: This bit is unmodified from upstream, but it's because it's an "EngineSetLauncher"

Comment (jdfreder) [inline ipyparallel/nbextension/clustermanager.py]: > unmodified from upstream

Ah okay

**Pull request merged**